cluster/addons/metrics-server/resource-reader.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
How this is handled for Heapster? It requires exactly the same permissions.

Reply:
See my comment below